tfx/orchestration/airflow/airflow_dag_runner.py

```diff
@@ -111,3 +114,26 @@ def run(self, tfx_pipeline: pipeline.Pipeline):
             component_impl_map[upstream_node])
 
     return airflow_dag
+
+
+  def _replace_runtime_params(self, comp):
+    for k, prop in comp.spec.exec_properties.copy().items():
+      if isinstance(prop, RuntimeParameter):
+        
+        # Airflow only supports string parameters
+        if prop.ptype != str:
+          raise RuntimeError(
+            f'RuntimeParameter in Airflow does not support {prop.ptype} only ptype supported is string'
+          )
+        
+        # If default is a template, we can just remove the template markers to make it work.
+        if prop.default.startswith('{{') and prop.default.endswith('}}'):
```

Comment:
Can you make this more descriptive like below?

```
# If default is a template, drop the template markers when inserting it into the .get() default argument below. Otherwise, provide the default as a quoted string.
```